.github/workflows/test.yml

```diff
@@ -126,10 +128,6 @@ jobs:
           CWD: "./packages/test-commerce-sdk-react"
           TARGET: commerce-sdk-react
 
-      - name: Check Repository Clean
-        if: env.IS_NOT_FORK == 'true' && env.IS_DEFAULT_NPM == 'true'
-        uses: "./.github/actions/check_clean"
```

Comment:
Bonus: before the publishing step, we used to have this step that checks whether the repo is clean. It's no longer necessary because the publishing step already does the check.